adapta/storage/delta_lake/README.md

```diff
@@ -26,11 +27,22 @@ adls_path = AdlsGen2Path.from_hdfs_path('abfss://container@account.dfs.core.wind
 # get Iterable[pandas.DataFrame]
 batches = load(azure_client, adls_path, batch_size=1000)
 
-# create a filter and apply it
-filter = (pyarrow_field("my_column") == "some-value")  # case sensitive!
-filtered = load(azure_client, adls_path, row_filter=filter, columns=["my_column", "my_other_column"])
+# create generic filter expressions
```

Comment:
I'd suggest extracting this into a separate readme section similar to astra docs